tsconfig.json

```diff
@@ -13,9 +13,7 @@
     "emitDecoratorMetadata": true,
     "allowJs": false,
     "outDir": "lib",
-    "inlineSourceMap": true,
-    "incremental": true,
-    "tsBuildInfoFile": "./.tsbuildinfo"
+    "inlineSourceMap": true
```

Comment:
*[Cross Post from #311 just for context]*

im sorry to hear that the test broke for you, but if you are using the lastest master branch (or pr #318) this shouldn't be a problem, because:

- test's are executed with ts-node, and not compiled
- rimraf is removed from the test script

i just added tsbuildinfo because in large projects it should remove time from compiling and in small should not be adding time.

@nodkz @Ben305 before merge: please test if this error is still a problem, because the testing changed in the (merged) pr #318 to be executed with ts-node (& not compiled)

Reply:
@hasezoey I met with another problem on tests:
- now I got an error with naming of test databases

<img width="866" alt="Screen Shot 2019-07-12 at 4 00 18 PM" src="https://user-images.githubusercontent.com/1946920/61120177-2996c900-a4be-11e9-9aec-b05d7700566f.png">


So I remove all your configurations for MongoDB and replace them with [mongodb-memory-server](https://github.com/nodkz/mongodb-memory-server). This is my lib exactly for testing purposes. Works almost everywhere, no need in configuration, no need in preinstalled mongo on dev machine.

Reply:
I reverted back your incremental changes in `tsconfig` and fixed a problem with MongoDB